plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -169,6 +171,33 @@ class ServiceNetworkingConfigSection extends ServiceConfigBaseSectionDisplay {
               });
             }
 
+            const containsServicePorts = portDefinitions.filter(function(
+              portDefinition
+            ) {
+              const result = portDefinition.servicePort
```

Comment:
won't work because `portDefinition.servicePort` is undefined instead of null, the reason I went with the ternary checking for `portDefinition.servicePort` first is to address both cases `null` `undefined`.